config/development.json

```diff
@@ -0,0 +1,6 @@
+{
+  "provider": "http://localhost:8545/rpc",
+  "address": "0x9e2d04eef5b16CFfB4328Ddd027B55736407B275",
+  "privateKey": "399b141d0cc2b863b2f514ffe53edc6afc9416d5899da4d9bd2350074c38f1c6",
+  "registryAddress": "0xB082D0cB94aefB147E7910CA579209f47c694C91"
```

Comment:
IF I understand correctly, this is the configuration used by the developer when playing wiht the app locally against a test chain. in that case, typically no contract will be deployed - and so we do not have the registyr address available. And so this line shoud be removed